extensions/ql-vscode/package.json

```diff
@@ -27,9 +27,9 @@
     "onView:codeQLQueryHistory",
     "onView:test-explorer",
     "onCommand:codeQL.checkForUpdatesToCLI",
-    "onCommand:codeQL.chooseDatabaseFolder",
-    "onCommand:codeQL.chooseDatabaseArchive",
-    "onCommand:codeQL.chooseDatabaseInternet",
+    "onCommand:codeQLDatabases.chooseDatabaseFolder",
```

Comment:
I had thought the difference between the `codeQL` namespace and the `codeQLDatabases` namespace was that the latter accepted an existing database as an argument (and so was only applicable in the databases view). Whereas the former could be invoked in any context.

I see you're disabling them as global commands, so maybe that's what we want.

Reply:
Happy to be persuaded about choice of convention, but my intent so far about the `codeQL$FOO.$BAR` name structure (where `$FOO` is nonempty) was just about scoping to portions of the interface. Consider for example `codeQLDatabases.sortByName` which does not take a database as an argument, and things like `codeQLQueryHistory.openQuery` which are relevant to user events taking place in the query history view, but don't imply any particular constraints about the types of those handlers.